packages/@guardian/source-react-components/src/icons/SvgSpinner.tsx

```diff
@@ -0,0 +1,61 @@
+import type { SerializedStyles } from '@emotion/react';
+import { css } from '@emotion/react';
+import type { EmotionJSX } from '@emotion/react/types/jsx-namespace';
+import { brand, iconSize } from '@guardian/source-foundations';
+import type { IconProps } from './types';
+
+interface LoadingCircleIconProps extends IconProps {
+	additionalCss?: SerializedStyles;
+}
+
+const circleLineThickness = 50;
+
+const lightblueStyles = css`
+	stroke: #a5c0e8;
+	stroke-width: ${circleLineThickness};
+	fill: transparent;
+`;
+const darkblueStyles = css`
+	stroke: ${brand['400']};
+	stroke-dasharray: 820;
+	stroke-dashoffset: 820;
+	stroke-width: ${circleLineThickness};
+	fill: transparent;
+`;
+
+export const SvgSpinner = ({
+	size,
+	additionalCss,
```

Comment:
What is a use case for this prop?

Reply:
I think it was used in manage-frontend, but can be removed in Source and added back if needed in the future